### ISSUE TITLE
BlurViews make the whole app turn white in Release build - Android

### ISSUE THREAD

**Issue by trucdongtxtv** [Kureev/react-native-blur#486]
**Steps to reproduce**
I created multiple BlurView's inside a screen component. I also did set `overlayColor` to `'transparent'` wrap the BlurView with another view and set {overflow: 'hidden'}.
     
`
   <View style={{ ...style, overflow: 'hidden' }}>

      <RNBlur
        blurType="light"
        blurRadius={20}
        overlayColor={'transparent'}
        style={{
          position: 'absolute',
          height: '100%',
          width: '100%',
          backgroundColor: 'transparent',
        }}
      />
      {children}
      </View>`

**Actual result**
The whole screen turns white. In some cases, it is white-faded.

**Expected result**
No white fades exist, and the BlurViews are bounded, aligned and shown as normally configured.

**Comment by hainguyentien** [comment]
@trucdongtxtv  Did you found any workaround yet?

**Comment by trucdongtxtv** [comment]
> @trucdongtxtv Did you find any workaround yet?

I tried setting overlay color to Color.TRANSPARENT in BlurViewManager.java. It doesn't work.

```
    @ReactProp(name = "overlayColor", customType = "Color")
    public void setColor(BlurView view, int color) {
        view.setOverlayColor(Color.TRANSPARENT);
        view.invalidate();
    }
```

**Comment by RahulMore-Codes** [comment]
@trucdongtxtv Yes exactly! Its working fine on Android-12 in both debug & release, but in Android-11 & below versions its showing white screen on debug & release both mode.

**Comment by Takesizmail** [comment]
+1

**Comment by godlikeAlex** [comment]
+1

**Comment by nguyen95** [comment]
+1

**Comment by nguyen95** [comment]
Same for me! Debug work fine but release show a matte overlay on screen

**Comment by remigyejacquot** [comment]
Same here! Only one screen is concerned. We implement the blur view in some part of the app and only the one which display a camera view not working. We display a modal with a blur as a background.
This issue does not prevent app to run but this is not the behavior we want.

**Comment by wragony** [comment]
I facing the same issue

**Comment by SaraChicaD** [comment]
Any updates here? Having the same issue.

**Comment by leonardoballand** [comment]
Same here

**Comment by SHYDEV-KR** [comment]
Any Updates?

**Comment by aurangs7** [comment]
still any update?
